packages/@aws-cdk/integ-tests/lib/test-case.ts

```diff
@@ -0,0 +1,48 @@
+import * as path from 'path';
+import { IntegManifest, TestCase } from '@aws-cdk/cloud-assembly-schema';
+import { attachCustomSynthesis, ISynthesisSession, Stack, StackProps } from '@aws-cdk/core';
+import * as fs from 'fs-extra';
+import { IntegManifestWriter } from './manifest-writer';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+export interface IntegTestCaseProps extends StackProps, TestCase {
+  testFileName: string;
+}
+
+// TODO Find a better name for this class
+class IntegTestCaseConstruct extends Construct {
```

Comment:
What if we just use this instead of the `IntegTestCase`? 

```ts
const integTest = new IntegTestCaseConstruct({
  testCases: [{
    stacks: [],
  }],
});
// or
integTest.addTestCase({
  stacks: [
    new MyStack(),
  ],
});
```

The `IntegTestCaseConstruct` would then have all the information necessary to synth the integ manifest.

Reply:
Well, a stack will have to be created one way or another. If we only use the construct, users would have to create it themselves:

```ts
const app = new App();
const stack = new Stack(app, 'stack');
const integTest = new IntegTestCaseConstruct(stack, 'test', ...);
```

which might be fine. But I wonder if we should simplify it for them.

Reply:
I think creating your own stacks is fine. What if we need to create a test case with multiple stacks?